plugins/msg-node/mcollective/src/openshift.rb

```diff
@@ -157,7 +157,12 @@ def cartridge_do_action
         exitcode, output, addtl_params = execute_action(action, args)
 
         if args['--with-container-uuid']
-          report_quota(output, args['--with-container-uuid'])
+          if action.to_s.include?("post-configure") || action.to_s.include?("deconfigure")
+            report_quota(output, args['--with-container-uuid'])
+          else
+            # do not print quota warning (output) during action "configure", to avoid double warning msg
+            report_quota('', args['--with-container-uuid'])
```

Comment:
If we aren't going to do anything with the output, why run `report_quota` at all? It doesn't appear to do anything other than generate output, so it seems needless to run it in this case.

Also, to ensure that quota is reported with everything but the 'configure' step, why not use `if action.to_s != "configure"` ?